accounts/keystore/watch.go

```diff
@@ -33,12 +34,14 @@ type watcher struct {
 	runEnded bool // set to true when runloop ends
 	starting bool // set to true prior to runloop starting
 	quit     chan struct{}
+	wg       sync.WaitGroup // wait for watcher loop to start
 }
 
 func newWatcher(ac *accountCache) *watcher {
 	return &watcher{
 		ac:   ac,
 		quit: make(chan struct{}),
+		wg:   sync.WaitGroup{},
```

Comment:
We typically use `wg` as the name for the WaitGroup tracking the shutdown of component goroutines. Since this one tracks something else, please rename it to `startupWG`.

Reply:
Thanks for the feedback.
I've updated the code as per your suggestion.